whelktool/scripts/cleanups/2020/01/normalize-unicode.groovy

```diff
@@ -1,10 +1,10 @@
-import java.text.Normalizer
+import whelk.util.Unicode
 
 PrintWriter failedIDs = getReportWriter("failed-IDs")
 PrintWriter scheduledForUpdate = getReportWriter("scheduled-for-update")
 
 selectBySqlWhere("deleted is false", silent: false, { docItem ->
-    if (!Normalizer.isNormalized(docItem.doc.getDataAsString(), Normalizer.Form.NFC)) {
+    if (!Unicode.isNormalized(docItem.doc.getDataAsString())) {
         scheduledForUpdate.println("${docItem.doc.getURI()}")
         docItem.scheduleSave()
```

Comment:
I *presume* we want to version this change (as is done here)? How many documents were affected?

Reply:
Lost network connection during dry-run. Rerunning now.